thoth/package_extract/image.py

```diff
@@ -468,9 +468,21 @@ def _get_python_interpreters(path: str) -> dict:
     result = []
 
     for py_path in glob.glob('{}/usr/bin/python*'.format(path)):
+        path_ = py_path[len(path):]
```

Comment:
Haven't tried this, but isn't this always asking system interpreter version?

Reply:
`path_` will have something like:
```
/usr/bin/python3.6-config
/usr/bin/python3-mako-render
/usr/bin/python2
/usr/bin/python3.6m-config
/usr/bin/python3
/usr/bin/python3.6m
/usr/bin/python3.7m-x86_64-config
/usr/bin/python3.7-config
/usr/bin/python3.7m
/usr/bin/python2.7
/usr/bin/python3.7
/usr/bin/python3-coverage
/usr/bin/python3.6
/usr/bin/python3.6m-x86_64-config
/usr/bin/python3.7m-config
/usr/bin/python3-config
/usr/bin/python
```

Reply:
Yes - that's what I was referring to. then in the line bellow:

```python
            line = run_command('{!r} --version'.format(path_), timeout=2).stdout
```

you will call python which is present in `/usr/bin/`, but not the one which is present in `{path}/usr/bin/` - in other words you will execute Python which is present in thoth's package-extract, not the one which is inside the analyzed container image.

Reply:
:smile: Yes, I see what you mean, thanks